src/StaticFAQ/index.js

```diff
@@ -74,13 +81,24 @@ class StaticFAQ extends React.Component<Props, State> {
           </Typography>
         </div>
         <div className="static-faq-body">
-          <Input
-            value={this.state.value}
-            onChange={this.handleSearchChange}
-            placeholder="What can we help you with?"
-            icon={<Magnify />}
-          />
-          <FAQCategoryList categoryId={categoryId} />
+          {!categoryId ? (
+            <React.Fragment>
```

Comment:
these conditions look already too complicated, what about e.g.

`{!categoryId && this.renderInput()}`